gravitee-apim-console-webui/src/components/alerts/alerts.html

```diff
@@ -15,8 +15,12 @@
     limitations under the License.
 
 -->
-<div class="gv-sub-content">
-  <div class="gv-forms gv-forms-fluid" layout="column">
+<div class="gv-sub-content" ng-class="{'ae-promo-content': $ctrl.isAlertEngineEnabled() == false}">
+  <div ng-if="$ctrl.isAlertEngineEnabled() == false">
+    <alert-engine-promotion />
+  </div>
+
+  <div ng-if="$ctrl.isAlertEngineEnabled() == true" class="gv-forms gv-forms-fluid" layout="column">
```

Comment:
Maybe we could have a top-level div with the condition `$ctrl.isAlertEngineEnabled() == true` ?
That would avoid repeating this condition in every next div

Reply:
done, removed from 2 divs and added them in a common div. One of the condition is still remains as it is a separate div.